spark/src/main/scala/geotrellis/spark/io/avro/codecs/ProjectedExtentCodec.scala

```diff
@@ -0,0 +1,38 @@
+package geotrellis.spark.io.avro.codecs
+
+import geotrellis.proj4.CRS
+import geotrellis.spark._
+import geotrellis.spark.io.avro._
+import geotrellis.spark.io.avro.codecs._
+import geotrellis.spark.io.avro.codecs.Implicits._
+import geotrellis.vector._
+
+import org.apache.avro._
+import org.apache.avro.generic._
+
+// --- //
+
+trait ProjectedExtentCodec {
+  implicit def projectedExtentCodec = new AvroRecordCodec[ProjectedExtent] {
+    def schema: Schema = SchemaBuilder
+      .record("ProjectedExtent").namespace("geotrellis.vector")
+      .fields()
+      .name("extent").`type`(extentCodec.schema).noDefault()
+      .name("epsg").`type`().intType().noDefault()
```

Comment:
Can you update to use the proj4 string.
Or, if possible, check if it's an EPSG code, and then fall back to CRS?